### ISSUE TITLE
Front: master layout

### ISSUE THREAD

**Issue by naui95** [InvoicePlane/InvoicePlane-v2#12]

**Comment by nielsdrost7** [comment]
## SMART Plan — Front — master layout epic status

> **Context/Epic**: Core
>
> This is an epic/tracking issue. Its sub-issue checklist needs a correction up front: none of the linked numbers (`#2`, `#10`, `#11`, `#22`, `#23`, `#24`, `#25`) correspond to the sub-features they're labelled with in this tracker. `#2` is "[Core]: Features & Improvements collection (v1 community)" (an unrelated wishlist), `#10` is "[Infrastructure]: Documentation efforts tracker" (an unrelated epic), `#11` is "[Core]: Login screen" (auth, not mobile navigation), `#24` is "[Projects]: change dropdown client menu..." (an unrelated Projects issue), and `#22`, `#23`, `#25` don't exist in this repository at all (this tracker only goes up to #606 with many gaps; those three numbers were never issued). The Measurable section below therefore tracks status by **capability**, grounded in the actual panel provider code, rather than by the stale issue numbers.

### Specific
Establish the shared Filament layout shell (navigation, sidebar, top bar, breadcrumbs, theming, footer) used by all three panels (`admin`, `company`, `user`).

### Measurable
- [x] **Dark-mode / Nord theme wiring** — done, and already applied to all three panels, not just company: `Modules/Core/Providers/AdminPanelProvider.php`, `CompanyPanelProvider.php`, and `UserPanelProvider.php` each call `->viteTheme('resources/css/filament/company/nord.css')`.
- [x] **Sidebar collapse behaviour** — done: all three panel providers call `->sidebarCollapsibleOnDesktop()`.
- [x] Three-panel shell exists — `AdminPanelProvider`, `CompanyPanelProvider`, `UserPanelProvider` are real, registered, and resource-discovering per `CLAUDE.md`.
- [ ] **Breadcrumb component** — not custom-built; whatever appears today is Filament v4's own default breadcrumb rendering. No dedicated breadcrumb component/override was found under `Modules/Core/Filament`.
- [ ] **Top-bar: tenant label, notification bell, account dropdown** — partially contradicted by current config: `CompanyPanelProvider` explicitly calls `->tenantMenu(false)`, disabling Filament's built-in tenant menu, and no notification bell or custom account dropdown component was found. The avatar piece of this depends on #399 (client & user avatars), which found the Gravatar driver unwired.
- [ ] **Responsive mobile navigation / hamburger drawer** — no dedicated Livewire component found; relies entirely on Filament's built-in responsive sidebar behaviour (untested here for whether it meets the "keyboard-accessible open/close" bar the epic body sets).
- [ ] **Footer scaffold (version badge, support link)** — no footer component or render hook found anywhere in `Modules/Core/Providers` or `resources/views`.
- Related, real, currently-open issue in this exact area (not one of the epic's own — but genuinely relevant): **#568** "Active sidebar navigation item is unreadable due to light background" — a concrete sidebar-theming bug that should probably be folded into this epic's scope.

**Definition of done for this epic**: all capabilities above are checked off, with real Filament accessibility checks passing (per the epic's own acceptance criteria), and the epic body's sub-issue links are corrected to point at real, matching issues (or removed if no such issues exist).

### Achievable
1. First, correct or remove the epic body's sub-issue links (`#2`, `#10`, `#11`, `#22`, `#23`, `#24`, `#25`) — they currently misdirect anyone trying to navigate from this epic to its actual sub-work, and three of them point at issues that don't exist.
2. Theming and sidebar-collapse (the two items already done) need no further work beyond verifying they satisfy the epic's "no unstyled flash" and accessibility acceptance criteria under real browser testing.
3. Breadcrumbs likely need no custom build if Filament v4's defaults already satisfy the acceptance criteria — verify before scoping new work, since building a custom breadcrumb component would be wasted effort if the framework default is sufficient.
4. Top-bar work (tenant label, notification bell, account dropdown, avatar) should be scoped as one real, numbered sub-issue — it currently has no tracking at all under a correct number — and sequenced after #399 (avatars) if the avatar piece is included.
5. Footer scaffold and the mobile hamburger drawer are the two capabilities with zero existing code; each should get its own correctly-numbered sub-issue rather than relying on this epic's stale links.

### Relevant
The master layout is the visual foundation every other panel page builds on; the fact that its own sub-issue checklist doesn't point at real tracked work means progress on this epic is currently unmeasurable through the issue tracker itself — fixing the links is as important as building the remaining capabilities.

### Time-bound
No fixed date — re-review once the sub-issue links are corrected and each remaining capability (top bar, mobile drawer, footer) has its own real issue number to track against.
